components/select/template.rst

```diff
@@ -2,10 +2,10 @@ Template Select
 ===============
 
 .. seo::
-    :description: Instructions for setting up template selects with ESPHome.
+    :description: Instructions for setting up Template Select(s) with ESPHome.
     :image: description.png
 
-The ``template`` select platform allows you to create a select with templated values
+The ``Template Select`` platform allows you to create a Select with templated values
```

Comment:
The ``template`` revered to the key of the platform, which now became invalid

Reply:
Opps, thanks @frenck